time timeStamp
float32 lat
float32 lon
float32 vel
float32 frontDistance
